acceptance/lib/helper.rb

```diff
@@ -0,0 +1 @@
+$LOAD_PATH << File.expand_path(File.dirname(__FILE__))
```

Comment:
with :load_path as an option in the beaker options file, one can eliminate this file and its usage which is largely deprecated in beaker.